corehq/apps/data_dictionary/templates/data_dictionary/partials/confirmation_modals.html

```diff
@@ -39,7 +39,7 @@ <h4 class="modal-title">
             </ul>
             <p>
               For more information on deprecated case types, see the
-              <a target="_blank" href="https://confluence.dimagi.com/display/commcarepublic/Deprecating+Case+Types+and+Case+Properties+using+Data+Dictionary">following documentation</a>.
+              <a target="_blank" href="https://dimagi.atlassian.net/wiki/spaces/commcarepublic/pages/2143944977/Data+Dictionary">following documentation</a>.
```

Comment:
More specific: https://dimagi.atlassian.net/wiki/spaces/commcarepublic/pages/2143944977/Data+Dictionary#Deprecating-%26-Delete-Case-Types-and-Case-Properties